components/partials/common/market-selection/markets-table-row.vue

```diff
@@ -84,18 +88,17 @@ export default Vue.extend({
     },
 
     isFavourite(): boolean {
-      // - todo implement business logic
+      const { accountFavouriteMarkets, market } = this
 
-      return false
+      return accountFavouriteMarkets.includes(market.marketId)
     }
   },
 
   methods: {
     updateWatchList() {
       const { market } = this
 
-      // - todo implement business logic
-      console.log(market.marketId)
+      this.$accessor.account.updateAccountFavouriteMarkets(market.marketId)
```

Comment:
Should be stored within `app.userState`, `account` is related to the trading account (subaccount).